New post: Creating backups (#45)
FalixNodes-Software/help-center: _posts/minecraft/general/2021-08-29-creating-backups.md

---
layout: post

> title:  "Creating backups"

Reviewer: ```suggestion
title:  "Creating Backups"
```

categories: Minecraft
tags: mcgeneral
permalink: /minecraft/general/creating-backups/
---

> # How to create backups

Reviewer: ```suggestion
# How To Create Backups
```

> 

Reviewer: Maybe add a short introduction to this article. I've come up with this:

Backing your server up is a very crucial task when you're a server owner. If anything bad happens to your server, such as griefing or/and unintended actions are performed by you or one of your server operators, you'll be able to revert those changes in your server. In this article, we'll give you a quick and easy explanation on how to create backups for your server within the Game Panel.

1. Make sure you have allocated backups to your server, you can do so in the [client page](https://client.falixnodes.net).

> 2. Head over to the [panel](https://panel.falixnodes.net).

Reviewer: ```suggestion
2. Head over to the [game panel](https://panel.falixnodes.net).
```

3. Click on your server.
4. Click on "Backups" at the top of the page.
5. Click on "Create backup" at the bottom right of the page.
6. Insert a name into the "Backup name".

> 7. You can exclude any files/folders by inserting their name into the "Ignored files & directories" field, you can also use wildcards, refer to the table below to learn how.

Reviewer: ```suggestion
7. If needed, you can exclude any files/folders by inserting their name into the "Ignored files & directories" field, you can also use wildcards, refer to the table below to learn how.
```

| Wildcard                   | Function                          |Example                                           |
|----------------------------|-----------------------------------|--------------------------------------------------|
| *                          | Replaces any number of characters |`*.jar` excludes all .jar files
`/home/container/plugins/*` excludes all folders/files in the "plugins" directory
`*ava` excludes all fles/folders that ends with "ava"
|----------------------------|-----------------------------------|--------------------------------------------------|
| ?                          | Replaces a single character       | F`?`ile  = F`o`le / F`i`le / F`a`le / F`e`le / F`u`le |
|----------------------------|-----------------------------------|--------------------------------------------------|
| []                         | Specifies a set of characters to match, `-` specifies a range of consecutive characters| `[Jj]`ava = `J`ava / `j`ava 
folder`[1-3]` = folder1 / folder2 / folder3 |

> > Note: Make sure to type every path/directory/filename/folder name on a separate line

Reviewer: ```suggestion
> Note: Make sure to type every path, directory, filename or folder name on a separate line
```

> 8. You can choose to lock the backup or leave it unlocked, locking it will prevent the backup from being deleted by a user, you can always unlock it later.

Reviewer: ```suggestion
8. You can choose to lock the backup or leave it unlocked, locking it will prevent the backup from being unintentionally deleted by a user, you can always unlock it later.
```

9. Your backup has been created!

> > Note: You can delete, restore, lock and even download your backup by clicking on the 3 dots to your right!

Reviewer: ```suggestion
> Note: You can also delete, restore and download the backup to your device by clicking on the 3 dots to your right!
```

Author: what about locking / unlocking?

Reviewer: You already mentioned locking and unlocking on step 8.

---
layout: post
title:  "Creating Backups"
categories: Minecraft
tags: mcgeneral
permalink: /minecraft/general/creating-backups/
---

# How To Create Backups

> Backing up your server is a crucial task when you're a server owner. If anything bad happens to your server, such as griefing and/or unintentional actions, you'll be able to revert those changes in your server. In this article, we'll give you a quick and easy explanation of how to create backups for your server within the Game Panel.

Reviewer 2: ```suggestion
Backing up your server is a crucial task when you're a server owner. If anything bad happens to your server, such as griefing and/or unintentional actions, you'll be able to revert those changes. 
In this article, we'll give you a quick and easy explanation of how to create backups for your server within the Game Panel.
```

> Backing your server up is a very crucial task when you're a server owner. If anything bad happens to your server, such as griefing or/and unintended actions are performed by you or one of your server operators, you'll be able to revert those changes in your server. In this article, we'll give you a quick and easy explanation on how to create backups for your server within the Game Panel.

Reviewer 2: ```suggestion
Backing up your server is a crucial task when you're a server owner. If anything bad happens to your server, such as griefing and/or unintentional actions, you'll be able to revert those changes in your server. In this article, we'll give you a quick and easy explanation of how to create backups for your server within the Game Panel.
```

Author: > easy explanation "of" how to create backups for your server within the Game Panel. 

"of" does not sit right with me, "on" is correct in my opinion

Reviewer 2: That's not what Grammarly says, and I agree with using "of". But really, "on" & "of" can both be used here.

@itsjuoum Your opinion?

Reviewer: When I wrote this and then checked it on Grammarly, I ignored the fact that it should be "of" as "on" sounded good as well. But since Grammarly says that "on" is grammatically incorrect, I think we should change to "of".

Reviewer 2: 👍 

@Mocab Please change it to "of".

> 1. Make sure you have allocated backups to your server, you can do so in the [client page](https://client.falixnodes.net).

Reviewer 2: ```suggestion
1. Make sure you have allocated backups to your server, you can do so in the [client panel](https://client.falixnodes.net).
```

2. Head over to the [game panel](https://panel.falixnodes.net).
3. Click on your server.
4. Click on "Backups" at the top of the page.
5. Click on "Create backup" at the bottom right of the page.

> 6. Insert a name into the "Backup name".

Reviewer 2: ```suggestion
6. Insert a name into the "Backup name" field.
```

|----------------------------|-----------------------------------|----------------------------------------------------------------------------------|
| *                          | Replaces any number of characters |• `*.jar` excludes all .jar files 						    |
|                            |                                   |• `/home/container/plugins/*` excludes all folders/files in the "plugins" directory |
|			     |					 |• `*ava` excludes all fles/folders that ends with "ava" 			    |
|----------------------------|-----------------------------------|----------------------------------------------------------------------------------|
| ?                          | Replaces a single character       | F`?`ile  = F`o`le / F`i`le / F`a`le / F`e`le / F`u`le 			    |
|----------------------------|-----------------------------------|----------------------------------------------------------------------------------|
| []                         | Specifies a set of characters to match, `-` specifies a range of consecutive characters |• `[Jj]`ava = `J`ava / `j`ava 
|			     |				 	 |• folder`[1-3]` = folder1 / folder2 / folder3 |

> > Note: Make sure to type every path, directory, filename or folder name on a separate line

Reviewer 2: ```suggestion
> Note: Make sure to type every path, directory, filename or folder name on a separate line.
```

8. You can choose to lock the backup, locking it will prevent the backup from being unintentionally deleted by a user, you can always unlock it later.

> 9. Your backup has been created!

Reviewer 2: ```suggestion

That's it! Your backup has been created!
```

> 8. You can choose to lock the backup or leave it unlocked, locking it will prevent the backup from being unintentionally deleted by a user, you can always unlock it later.

Reviewer 2: ```suggestion
8. You can choose to lock the backup, locking it will prevent the backup from being unintentionally deleted by a user, you can always unlock it later.
```

> > Note: You can also delete, restore and download the backup to your device by clicking on the 3 dots to your right!

Reviewer 2: ```suggestion
> Note: You can also delete, restore and download the backup to your device by clicking the 3 dots to the right of the backup!
```
